src/psnawp_api/models/user.py

```diff
@@ -250,6 +250,75 @@ def is_blocked(self) -> bool:
         ).json()
         return self.account_id in response["blockList"]
 
+    def get_all_trophies(self, limit=799):

```

Comment:
I think the max throphies returned are 100. You can probably pass an offset aswell to support a way to get all thropies.

Reply:
i've tested it and believe the number of returned titles is much more. had success with profiles with 400+ titles